db/schema.rb

```diff
@@ -10,11 +10,30 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_11_16_110213) do
+ActiveRecord::Schema.define(version: 2021_11_23_095817) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "bookmarked_posts", force: :cascade do |t|
+    t.bigint "user_id", null: false
+    t.bigint "post_id", null: false
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["post_id"], name: "index_bookmarked_posts_on_post_id"
+    t.index ["user_id"], name: "index_bookmarked_posts_on_user_id"
+  end
+
+  create_table "comments", force: :cascade do |t|
```

Comment:
Redzi, Rails automatiski uzgenereja no tavas datubazes schema tabulu.

Bet comments neeksiste šajā brančā. Ar to tev vajag būt ļoti uzmanigam, kad strādāsi pie vairakam fičam vienlaikus - tev jāseko schema.rb failām. Vai sākt darbu ar tiro datubazi.

Reply:
Es parasti uztaisu pa jaunam db, piemirsu laikam soreiz, ieslideja ari nepamanosi commita.